src/main/java/lotto/ui/OutputView.java

```diff
@@ -0,0 +1,63 @@
+package lotto.ui;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import lotto.Lotto;
+import lotto.Rank;
+
+public class OutputView {
+    private static final String LOTTO_QUANTITY_NOTIFY_MESSAGE = "%d개를 구매했습니다.";
+    private static final String WINNING_STATISTICS_MESSAGE = "당첨 통계";
+    private static final String NEW_LINE = "\n";
+    private static final String DASH = "-";
+    private static final String DUPLICATE_NOTIFY_MESSAGE = "%d개 일치";
+    private static final String BONUS_NOTIFY_MESSAGE = ", 보너스 볼 일치";
+    private static final String SPACE = " ";
+    private static final String PRIZE_MESSAGE = "(%,d원)";
+    private static final String WINNING_LOTTO_NUMBER_MESSAGE = "%d개";
+    private static final String EARNING_RATE_NOTIFY_MESSAGE = "총 수익률은 %.1f%%입니다.";
+    private static final String ERROR_PREFIX = "[ERROR] ";
+
+    public static void printLottoQuantity(int lottoQuantity) {
+        System.out.print(NEW_LINE);
+        System.out.println(String.format(LOTTO_QUANTITY_NOTIFY_MESSAGE, lottoQuantity));
+    }
+
+    public static void printLottos(List<Lotto> lottos) {
+        for (Lotto lotto : lottos) {
+            List<Integer> sortedLotto = new ArrayList<>(lotto.getNumbers());
+            sortLottoAscendingOrder(sortedLotto);
+            System.out.println(sortedLotto);
+        }
+    }
+
+    private static void sortLottoAscendingOrder(List<Integer> numbers) {
+        Collections.sort(numbers);
+    }
+
+    public static void printStatistics(Map<Rank, Integer> finalResult) {
+        System.out.println(NEW_LINE + WINNING_STATISTICS_MESSAGE + NEW_LINE + DASH.repeat(3));
```

Comment:
(이렇게 바꿔보는 것도 좋을 것 같아요! 개인적인 생각이 많이 들어가 있어요:))

```suggestion
        System.out.println("""
        당첨 통계
        ---
        """);
```
개인적으로 너무 많은 문자열을 상수로 두는 부분이 오히려 가독성을 나쁘게 만든 것 같다고 생각합니다!
직관적으로 해당 텍스트를 그대로 사용 가능하다면 충분히 한줄로 표현해도 좋을 것 같아요 :)

Reply:
오,, 전혀 생각하지 못했던 부분이네요,, 훨씬 깔끔한 것 같아요 감사합니다!